views/partials/top-level/security-tips.hbs

```diff
@@ -18,60 +18,61 @@
           {{#each copy}}
-            {{#each leads}}
-              <p class="article-paragraph lead">
-                <span class="drop-cap">
-                  {{ ./../dropCap }}
-                </span>
-                <span>{{ this }}</span>
-              </p>
-            {{/each}}
+          {{#each leads}}
+          <p class="article-paragraph lead">
+            <span class="drop-cap">
+              {{ ./../dropCap }}
+            </span>
+            <span>{{ this }}</span>
+          </p>
+          {{/each}}
 
-            {{#if toggles}}
+              {{#if toggles}}
               {{#each this.toggles}}
-                {{#if securityTip}}
-                  {{> top-level/security-tips/security-tip-callout }}
-                {{/if}}
-                {{#if subhead}}
-                  <div class="toggle-parent inactive st-toggle-wrapper">
-                    <div class="toggle-info-wrapper">
-                      <button class="toggle svg-wrap article-toggle" aria-label="{{ getString "show-content" }}">
-                        <span class="article-subhead toggle-subhead">{{{ subhead }}}</span>                    
-                        {{> svg/toggle-down }}
-                      </button>
-                    </div>
-                    <div class="toggle-child">
-                      {{> top-level/security-tips/tip-content }}
-                    </div><!--closes toggle-child-->
-                  </div>
-                {{/if}}
+              {{#if securityTip}}
+              {{> top-level/security-tips/security-tip-callout }}
+              {{/if}}
+              {{#if subhead}}
+              <div class="toggle-parent inactive st-toggle-wrapper">
+                <div class="toggle-info-wrapper">
+                  <button class="toggle svg-wrap article-toggle" aria-label="{{ getString " show-content" }}">
+                    <span class="article-subhead toggle-subhead">{{{ subhead }}}</span>
+                    {{> svg/toggle-down }}
+                  </button>
+                </div>
+                <div class="toggle-child">
+                  {{> top-level/security-tips/tip-content }}
+                </div>
+                <!--closes toggle-child-->
+              </div>
+              {{/if}}
               {{/each}}
-            {{/if}}
-            {{#if subhead}}
+              {{/if}}
+              {{#if subhead}}
               <h3 class="article-subhead">{{{ subhead }}}</h3>
-            {{/if}}
-            {{> top-level/security-tips/tip-content }}
-      
-            {{> top-level/security-tips/dos-and-donts }}
-          {{/each}}
-        </div>
-      {{/each}}
-    </div>
-  </div>
-</section>
+              {{/if}}
+              {{> top-level/security-tips/tip-content }}
+              
+              {{> top-level/security-tips/dos-and-donts }}
+              {{/each}}
+              </div>
+              {{/each}}
+              </div>
+              </div>
+              </section>
```

Comment:
This seems to have flattened out a bunch of the indentation? Is that on purpose?

Reply:
fixed.